modules/adagioBidAdapter.md

```diff
@@ -37,7 +37,56 @@ Connects to Adagio demand source to fetch bids.
             environment: 'mobile', // Recommended. Environment where the page is displayed.
             category: 'sport', // Recommended. Category of the content displayed in the page.
             subcategory: 'handball', // Optional. Subcategory of the content displayed in the page.
-            postBid: false // Optional. Use it in case of Post-bid integration only.
+            postBid: false, // Optional. Use it in case of Post-bid integration only.
+            // Optional debug mode, used to get a bid response with expected cpm.
+            debug: {
+              enabled: true,
+              cpm: 3.00 // default to 1.00
+            }
+          }
+        }]
+      },
+      {
+        code: 'article_outstream',
+        mediaTypes: {
+          video: {
+            outstream: {
```

Comment:
this `outstream: {` is not needed. The correct format should be: 

```
{
  code: "video1",
    mediaTypes: {
      video: {
        context: "outstream",
        playerSize: [640, 480],
        ...
      }
   }
}
``` 

Reply:
Thanks to take the time to fully test our PR.

I think the issue is due to the default Google slot creative template and some error in the test page.

To see the renderer playing, please update the test page with the code bellow:

```js
var adUnits = [
  {
    code: "video1",
    bids: [
    {
      bidder: "adagio",
      params: {
        organizationId: "1002",
        site: "adagio-io",
        placement: "in_article",
        adUnitElementId: "video1", // --> important for our renderer
        …
    }]
  }
];

function initAdserver(bidResponses) {
  if (pbjs.initAdserverSet) return;
  pbjs.initAdserverSet = true;
  googletag.cmd.push(function () {
    pbjs.que.push(function () {
      pbjs.setTargetingForGPTAsync('video1'); // Required to match the adUnit defined with googletag.defineSlot(), see bellow.
      console.log('bidResponses', bidResponses) // inspect the full bid response object
      googletag.pubads().refresh();
    });
  });
}

…
// Define a slot which can serve a prebid-universal-creative template.
var slot1 = googletag
  .defineSlot("/21686208299/pave", [[1, 1]], "video1")
  .addService(googletag.pubads());
```

<img width="714" alt="adagio-outstream" src="https://user-images.githubusercontent.com/16453882/101092482-06d8e900-35ba-11eb-9fae-c995e6cdd36b.png">

Reply:
Thank you, it works fine! 
If you want to update the adagioBidAdapter.md file and fix the example (by removing the `outstream: {` line (and the corresponding `}` ) we can merge this one.

Reply:
Hi @musikele 

I amended the adagioBidAdapter.md, so you can merge when you want :)